src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -428,22 +427,22 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, ControlValue
    */
   _handleKeydown(event: KeyboardEvent) {
     // Prevent propagation for all alphanumeric characters in order to avoid selection issues
-    if ((event.key && event.key.length === 1) ||
-      (event.keyCode >= A && event.keyCode <= Z) ||
-      (event.keyCode >= ZERO && event.keyCode <= NINE) ||
-      (event.keyCode === SPACE)
-      || (this.preventHomeEndKeyPropagation && (event.keyCode === HOME || event.keyCode === END))
+    if ((event.key && event.key.length === 1)
+      || (event.key >= 'A' && event.key <= 'Z')
```

Comment:
all of these single character comparisons should be removed, since we have 

https://github.com/bithost-gmbh/ngx-mat-select-search/blob/e668f98e59152e8cff72c5c40a520872f93ea064/src/app/mat-select-search/mat-select-search.component.ts#L431

Moreover, `(event.key >= 'A' && event.key <= 'Z')` would also match e.g. `'PrintScreen'`, since `'PrintScreen' >= 'A' && 'PrintScreen' <= 'Z' === true`, because of lexicographical comparison of strings, see https://tc39.es/ecma262/multipage/abstract-operations.html#sec-islessthan